packages/sku/src/program/commands/start/start.action.ts

```diff
@@ -13,6 +17,17 @@ export const startAction = async (
   command: Command,
 ) => {
   const { environment } = command.optsWithGlobals();
+
+  console.log(chalk.blue(`sku start`));
+
+  await Promise.all([
+    await configureProject(skuContext),
+    await watchVocabCompile(skuContext),
+  ]);
+
+  withHostile(checkHosts)(skuContext);
```

Comment:
`checkHosts`, just like `validatePeerDeps`, simply emit warning logs, so there's no need for them to be awaited.